src/Common/src/Interop/Unix/System.Native/Interop.Stat.cs

```diff
@@ -6,11 +6,11 @@
 
 internal static partial class Interop
 {
-    internal static partial class NativeIO
+    internal static partial class System
     {
-        internal struct FileStats
+        internal struct FileStatus
         {
-            private FileStatsFlags Flags;
+            internal FileStatusFlags Flags;
             internal int Mode;
             internal int Uid;
             internal int Gid;
```

Comment:
Yikes, I got the size of the int64_t fields wrong, but the only use so far doesn't read that far.